docs/e2e-tests.md

```diff
@@ -28,3 +28,21 @@ The [E2E test folder](../e2e-tests/web/test) contains two important subdirectori
 
 When writing new tests create helper classes for each Page you are using in the tests to keep the tests uniform and readable. This is also called the [page object pattern](https://webdriver.io/docs/pageobjects/).
 The tests itself are then created in the test folder and have to end on `*.e2e.ts`.
+
+### Native
+
+#### Selectors
+
+Selectors are used to select an element in the app programatically.
+However, during native development most of the common selectors are not available. 
+Therefore, you should use the accessibility identifier for this.
+Add the accessibility-id to a react component using `testID('Example-Component')`. You can query this component with `$('~Example-Component')` in your test.
+
+For more complex queries you should add/use a custom [Selector]('../e2e/native/Selector.ts) using [predicate strings](https://github.com/facebookarchive/WebDriverAgent/wiki/Predicate-Queries-Construction-Rules) for iOS and [UiSelectors](https://developer.android.com/reference/androidx/test/uiautomator/UiSelector) for Android.
```

Comment:
The `WebDriverAgent` repo is archived, is that intended?

Reply:
Actually no it isn't. But personally I liked that this guide is pretty short and only showed the relevant predicates. A full guide would can be found in the [Apple Documentation Archive](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Predicates/AdditionalChapters/Introduction.html). Would you prefer if I replace it?

Reply:
No, I guess it's fine to keep it. If you want, you can add apple as second resource, your choice!